ui/app/styles/components/services.scss

```diff
@@ -172,3 +172,21 @@ table.health-checks {
     }
   }
 }
+
+$tagPadding: 0.75em;
+
+.tag {
+  &.canary {
+    overflow: hidden;
+    &:before {
+      content: "Canary";
```

Comment:
Fun CSS trick to add copy to an element without adding it in the template itself.